source/MRMesh/MRVoxelsConversions.cpp

```diff
@@ -11,10 +11,24 @@
 #include "MRFastWindingNumber.h"
 #include "MRPch/MRTBB.h"
 #include "MRPch/MROpenvdb.h"
+
+#include <boost/mp11/algorithm.hpp>
```

Comment:
why do we need it?

Reply:
It was required for the compile-time parameters and their checks; it is reworked and removed now.